experimental/local_file_system_memory/readme.md

```diff
@@ -0,0 +1,72 @@
+# 🧠 GPT Memory
+
+LocalFilesystemMemory is a layer over Llama Index's GPTSimpleVectorIndex which abstracts away the details of managing indexes saved on disk. It assumes / creates a directory structure for storing indexes and provides a simple interface for managing them.
+
+## LocalFilesystemMemory
+
+1. Instantiate the memory with a folder where it will store its persistent memory.
+2. Create and name indexes. It will automatically generate a summary of the documents you provide.
+3. When you are ready, you can query the memory, and it will proceed in two steps. (a) It will figure out which of the named indexes is the right one to load into memory and query, based on how your query matches against the previously generated summaries. (b) It will then run that query against the most relevant index.
+
+An example using some Wikipedia files you can find in this repo:
+
+```python
+from llama_index import SimpleDirectoryReader
+from gpt_memory import LocalFilesystemMemory
+phil = SimpleDirectoryReader("gpt_memory/examples/phil")
+animals = SimpleDirectoryReader("gpt_memory/examples/animals")
+phil_docs = phil.load_data()
+animal_docs = animals.load_data()
+
+memory = LocalFilesystemMemory("example")
+memory.create_index("phil", phil_docs)
```

Comment:
maybe not for now, but I'd prefer decouple the memory from the index structure. That way, users can still specify GPTSimpleVectorIndex, or any index that they'd want, and they could call something like "LocalFileSystemMemory.save` and "LocalFileSystemMemory.load" on an index.

This makes the overall UX a bit more modular and will fit in with some high-level abstractions we're also building :)

Reply:
Hmm actually i may need to think about this, since you also have a "routing" mechanism within the query call that depends on the memory

Reply:
Yeah, happy to conform to whatever guidelines fit your overall vision. To be honest, it's not immediately clear to me yet, so I'll await further direction here. 

And yes, the routing is a fun part that I hope to use more of in the future.